src/view/Office.vue

```diff
@@ -24,23 +24,29 @@
 	<transition name="fade" appear>
 		<div class="office-viewer">
 			<div v-if="showLoadingIndicator"
-				id="office-viewer__loading-overlay"
+				class="office-viewer__loading-overlay"
 				:class="{ debug: debug }">
-				<NcEmptyContent v-if="!error" icon="icon-loading">
-					{{ t('richdocuments', 'Loading {filename} …', { filename: basename }, 1, {escape: false}) }}
-					<template #desc>
-						<button @click="close">
+				<NcEmptyContent v-if="!error" :title="t('richdocuments', 'Loading {filename} …', { filename: basename }, 1, {escape: false})">
+					<template #icon>
+						<NcLoadingIcon />
+					</template>
+					<template #action>
+						<NcButton @click="close">
 							{{ t('richdocuments', 'Cancel') }}
-						</button>
+						</NcButton>
 					</template>
 				</NcEmptyContent>
-				<NcEmptyContent v-else icon="icon-error">
-					{{ t('richdocuments', 'Document loading failed') }}
-					<template #desc>
-						{{ errorMessage }}<br><br>
-						<button @click="close">
-							{{ t('richdocuments', 'Close') }}
-						</button>
+				<NcEmptyContent v-else :title="t('richdocuments', 'Document loading failed')">
+					<template #icon>
+						<AlertOctagonOutline />
+					</template>
+					<template #action>
+						<p>{{ errorMessage }}</p>
```

Comment:
Any reason not to put this into a description?
https://github.com/nextcloud/nextcloud-vue/blob/stable7/src/components/NcEmptyContent/NcEmptyContent.vue#L96-L98

Reply:
No reason, but good point. Fixed.